src/main.rs

```diff
@@ -83,6 +87,10 @@ fn main() {
             ),
             Err(x) => handle_error(&x),
         },
+        CommandResult::Concatenate(x) => match x {
+            Ok(()) => handle_output(""),
```

Comment:
Will this just print out a newline? Perhaps we should just return `0`. Does the python version do anything or output anything after or is it silent?